### ISSUE TITLE
Noisy output at inference

### ISSUE THREAD

**Issue by cyiheng** [mobaidoctor/med-ddpm#48]
Hello, first I would thank you for your work !

I have some questions about how should I train my model to perform image-to-image translation task to generate CT.

I have around 200 images that I resized to 96x96x96, I trained for a total of 20 000 iterations to match 100 real epochs. As explained in your paper, the first 10k iterations was with a learning rate of 1e-5 and then with 1e-6. 

However, when trying to generate images with one source image, I have most of the time very noisy images (an example of 10 sample from one input). 

![Image](https://github.com/user-attachments/assets/a8fad7f1-31e6-4a80-89be-8cd233296c91)

I also used a timestep of 1000 as mentioned in a previous but still changed nothing.

The model seems to converge (maybe not enough ?) and even during training the sampled image are most of time noisy.

Do you have an idea about why this happens ? What kind of loss value do you have during your training ? Are the sampled image noisy as well ?

Thank you in advance.
